VMEncryption/main/check_util.py

```diff
@@ -99,6 +99,14 @@ def check_kek_url(self, test_kek_url, message):
             raise Exception('\n' + message + '\nActual: ' + test_kek_url + '\nExpected: ' + expected + "\n")
         return
 
+    def check_mhsm_url(self, test_mhsm_url, message):
+        """basic sanity check of the MHSM url"""
+        expected = "https://keyvault-name}.{vault-endpoint}"
+        pattern = re.compile(r'https://(.)+\\.(managedhsm)\\.(.)+(:443)?\\/keys/[^\\/]+\\/[0-9a-zA-Z]+$', re.IGNORECASE)
+        if not (test_mhsm_url and pattern.match(test_mhsm_url)):
+            raise Exception('\n' + message + '\nActual: ' + test_mhsm_url + '\nExpected: ' + expected + "\n")
```

Comment:
Why `test_mhsm_url and pattern.match(test_mhsm_url)` , can't we just have `pattern.match(test_mhsm_url)`?

Reply:
I followed other check conventions of kek and kv_url checks